cmd/task/list.go

```diff
@@ -30,12 +31,22 @@ func List(server, taskView, pageToken, stateFilter string, pageSize uint32, all
 		return err
 	}
 
+	tags := make(map[string]string)
+	for _, v := range tagsFilter {
+		parts := strings.Split(v, "=")
```

Comment:
This doesn't handle the case where a key or value contains an "=" sign. Do we care about that case?

Reply:
do we handle that case in `funnel run`? I took this pattern from there.